app/controllers/contestant_projects_controller.rb

```diff
@@ -0,0 +1,11 @@
+class ContestantProjectsController < ApplicationController
+
+def new
```

Comment:
Typically the `new` action is for displaying a form to create a new resource. What you want here instead is a `create` action since you're actually creating the joins record here.